src/Raven.Server/Indexing/LuceneVoronDirectory.cs

```diff
@@ -68,19 +68,18 @@ public override void TouchFile(string name)
             if (readResult == null)
                 throw new FileNotFoundException("Could not find file", name);
 
-            filesTree.Add(name, Stream.Null);
+            var pos = filesTree.DirectAdd(name, readResult.Reader.Length);
```

Comment:
That will require us to read/write the whole thing, and save it the journal.
We can have a separate value for that no?